OpenRA.Mods.Common/Traits/Render/LeavesTrails.cs

```diff
@@ -125,7 +125,7 @@ public void Tick(Actor self)
 				cachedFacing = facing != null ? facing.Facing : 0;
 				ticks = 0;
 
-				cachedInterval = isMoving && !wasStationary ? Info.MovingInterval : Info.StationaryInterval;
+				cachedInterval = isMoving ? Info.MovingInterval : Info.StationaryInterval;
```

Comment:
Yes. Yes, it does. Also when transitioning from moving to stationary as well, so I updated that check accordingly to that.